src/app/mapControls/FiltersContainer.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'app/config',
+    'app/mapControls/Filter',
+
+    'dijit/_TemplatedMixin',
+    'dijit/_WidgetBase',
+    'dijit/_WidgetsInTemplateMixin',
+
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+    'dojo/dom-construct',
+    'dojo/text!app/mapControls/templates/FiltersContainer.html',
+    'dojo/topic'
+], function (
+    config,
+    Filter,
+
+    _TemplatedMixin,
+    _WidgetBase,
+    _WidgetsInTemplateMixin,
+
+    declare,
+    lang,
+    domConstruct,
+    template,
+    topic
+) {
+    return declare([_WidgetBase, _TemplatedMixin, _WidgetsInTemplateMixin], {
+        // description:
+        //      Holds filter widgets and coordinates their work
+        templateString: template,
+        baseClass: 'filters-container',
+        widgetsInTemplate: true,
```

Comment:
I don't think you need any of this. you don't have any data-dojo-types in your template.